R/quit_if_no_pacta_relevant_data.R

```diff
@@ -0,0 +1,30 @@
+quit_if_no_pacta_relevant_data <- function(portfolio) {
```

Comment:
Should this function be documented?

Reply:
My gut feeling is that if the function belongs in its own dedicated file (ie. `quit_if_no_pacta_relevant_data.R`) then it should be documented, otherwise maybe just add it to the bottom of `utils.R`?

Reply:
Ah, actually just noticed that Alex mentioned changing the name to `user_error_checks.R`. That could also make sense.

Reply:
I'm not opposed to documenting it, but I am opposed to giving ANY suggestion at all that this function should be used by anyone, anywhere other than where I've precisely put it in this PR. Frankly, this is a dangerous function that is only necessary because of a very specific, peculiar behavior of the TM server that we need to play nice with.

On the other hand, I do want this to be/remain tested, and the unique R script for it follows the {testthat} mode of putting a function in it's own script, naming the script as the name of the function, and having a corollary {testthat} script in the tests directory. If there's a better/reasonable way to test it that doesn't conflict with standard tidyverse recommendation (I'm really trying to stick to it for your benefit), then I'm open to those suggestions.